File: 백준/Bronze/14681. 사분면 고르기/사분면 고르기.c
#include <stdio.h>

int main()
{
    int arr[2][2] = {
        {3, 2},
        {4, 1}
    };
    
    int x, y;
    scanf("%d\n%d", &x, &y);

    x = (x / abs(x) + 1) / 2;
    y = (y / abs(y) + 1) / 2;
    
    printf("%d", arr[x][y]);
}

File: 백준/Bronze/14681. 사분면 고르기/사분면 고르기.java
import java.util.*;
import java.lang.Math;

public class Main
{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int x, y;
		int [][] arr = {{3, 2}, {4, 1}};
		
		x = sc.nextInt();
		y = sc.nextInt();
		
		x = (x / Math.abs(x) + 1) / 2;
        y = (y / Math.abs(y) + 1) / 2;
		
		System.out.println(arr[x][y]);
	}
}